src/store/ducks/transaction/actionTransactions.ts

```diff
@@ -0,0 +1,91 @@
+import { takeEvery, call, put } from 'redux-saga/effects';
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { RootState } from 'store/types';
+import snxJSConnector from 'utils/snxJSConnector';
+import { fetchDelegateWalletInfoRequest } from '../delegates/delegateWalletInfo';
+
+export interface Transaction {
+	hash: string;
+}
+export interface TransactionError {
+	errorMessageKey: string;
+	// Id might be a hash, but if we failed before getting a hash another id string
+	id: string;
+}
+
+type ActionTransactionState = {
+	byHash: Record<string, Transaction>;
+	errorsById: Record<string, TransactionError>;
+};
+const initialState: ActionTransactionState = {
+	byHash: {},
+	errorsById: {},
+};
+
+const sliceName = 'actionTransactions';
+
+export const gasInfoSlice = createSlice({
```

Comment:
should be something like `actionTransactionsSlice`

Reply:
ops!